client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -40,12 +44,14 @@
 
     private static final long serialVersionUID = 1L;
     private static final ReaderGroupConfigSerializer SERIALIZER = new ReaderGroupConfigSerializer();
+    private static final String USE_DEFAULT_SCOPE = "_defaultScope";
```

Comment:
I'm not entirely convinced that using this constant is the right way to enforce the use a default scope. We should use the default scope when a stream is added to the configuration, and make sure that stream cuts are such that the streams carry their corresponding scopes.

Reply:
(answered as part of the first comment).